console_backend/src/ntrip_tab.rs

```diff
@@ -352,6 +336,7 @@ impl NtripState {
         msg_sender: MsgSender,
         mut heartbeat: Heartbeat,
         options: NtripOptions,
+        binary: String,
```

Comment:
to me this function arg is not the best name. And it doesn't make since for binary to be String, i'd expect it to be Vec<u8>

Reply:
and should it be an Option

Reply:
oops binary is probably not the best name,
maybe `converter_path` would be better